app/views/notifications_mailer/debrief_due.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Debrief due</h1>
+
+<p>It's time for your team to self-assess its progress.</p>
+
+<ol>
+  <li>Have one person open the <%= link_to 'team debrief form', new_project_iteration_debrief_url(@iteration.project, @iteration) %> and use it to guide a quick chat with team members.</li>
```

Comment:
Any considerations you think we need to make to avoid completion without the involvement of a mentor. Long term we don't want this behaviour to be dependent on a mentor but as we're currently delivering things could there be a risk of confusion?